reactioncommerce/reaction
# Fixes #2736: added unique to slug

Resolves #2736 
Impact: major
Type: bugfix

## Issue
Shops with the same name were not accessible

## Solution
Slug is already saved in the Shop Schema, we only need to check for its uniqueness.

## Testing
1. Invite a new user to marketplace
1. Sign in as the new user
1. Name your shop "Reaction2" (or anything)
1. Invite a second new user to marketplace
1. Sign in with your new user
1. Name your shop the same name as you did the previous new shop ("Reaction2")
1. See that both of the shops are available for the `owner` user.
1. Also as the 2nd user try to change the `slug` to a existing slug(reaction2)
1. Note the form won't allow you to do that.
